web/models.py

```
@@ -1752,6 +1753,62 @@ def display_name(self):
+
+    def clean(self) -> None:
+        super().clean()
+        # Validate that link is provided for online meetups
+        if self.event_type == "online" and not self.link:
+            raise ValidationError({"link": "Online meetups require a meeting link"})
+        # Validate that location is provided for in-person meetups
+        if self.event_type == "in_person" and not self.location:
+            raise ValidationError({"location": "In-person meetups require a location"})
+
+    def __str__(self) -> str:
+        return self.title
```

Ruff (0.8.2)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `__str__` method should come before custom method

(DJ012)